pages/k8s/1.16/components.md

```diff
@@ -0,0 +1,329 @@
+</tr>
+<tr>
+  <td> kata </td>
+  <td> Kata untrusted container runtime subordinate </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-kata">docs</a> </td> <td> <a href=""> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> keepalived </td>
+  <td> Failover and monitoring daemon for LVS clusters </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-keepalived">docs</a> </td> <td> <a href="https://github.com/juju-solutions/charm-keepalived"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> kubeapi-load-balancer </td>
+  <td> Nginx Load Balancer </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-kubeapi-load-balancer">docs</a> </td> <td> <a href="https://nginx.org/en/"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> kubernetes-master </td>
+  <td> The Kubernetes control plane. </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-kubernetes-master">docs</a> </td> <td> <a href="https://github.com/charmed-kubernetes/charm-kubernetes-master"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> kubernetes-worker </td>
+  <td> The workload bearing units of a kubernetes cluster </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-kubernetes-worker">docs</a> </td> <td> <a href="https://github.com/charmed-kubernetes/charm-kubernetes-worker"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> openstack-integrator </td>
+  <td> Proxy charm to enable OpenStack integrations via Juju relations. </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-openstack-integrator">docs</a> </td> <td> <a href="https://github.com/juju-solutions/charm-openstack-integrator"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> tigera-secure-ee </td>
+  <td> Tigera Secure Enterprise Edition </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-tigera-secure-ee">docs</a> </td> <td> <a href=""> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+<tr>
+  <td> vsphere-integrator </td>
+  <td> Proxy charm to enable VMware vSphere integrations via Juju relations. </td>
+  <td> <a href="/kubernetes/docs/1.16/charm-vsphere-integrator">docs</a> </td> <td> <a href="https://github.com/juju-solutions/charm-vsphere-integrator"> source </a> </td>
+  <td> <a href="https://bugs.launchpad.net/charmed-kubernetes"> bugs</a> </td>
+  <td> -- </td>
+  <td> -- </td>
+</tr>
+
+ </tbody>
+ </table>
+
+## Compatible Charms
```

Comment:
Will this section and Addons be manually edited?

Reply:
no, not really. The content will be autogenerated but the list it uses will have to be manually maintained (basically, it's just a list of the charms)